amostra/tests/conftest.py

```diff
@@ -0,0 +1,15 @@
+import amostra.mongo_client
+from pymongo import MongoClient
+import pytest
+
+
+connection = MongoClient('localhost', 27017)
+db = connection['tests-amostra']
```

Comment:
If we do this inside the fixture, then each test will get its own, clean collection to start with.

Also, we can ensure that each test cleans up after itself by using a finalizer. See this example, and consult the pytest documentation for more. https://github.com/bluesky/suitcase-mongo/blob/5f150dde5f3ec5f11aa0b2076113211f941dbb3f/suitcase/mongo_normalized/tests/fixtures.py#L16-L19

Reply:
Agree.